gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/services/CreateGraphService.java

```diff
@@ -56,6 +56,7 @@ private CRDCreateRequestBody buildCreateCRDRequestBody(final GaaSCreateRequestBo
                 .kind(kind)
```

Comment:
In the future you could convert to a `Gaffer` object as found in the `gaffer-controller` and move `gaffer-controller` the model to the `gaas-common`. This is a more generic option with a `LinkedHashMap` based `GraphSpec` rather than the more specialised `CRDCreateRequestBody`. At the end of the day both give the same JSON output